cuqi/mcmc/_pcn.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+import cuqi
+from cuqi.mcmc import SamplerNew
+from cuqi.array import CUQIarray
+
+class pCNNew(SamplerNew):  # Refactor to Proposal-based sampler?
+
+    def __init__(self, target, scale=1.0, **kwargs):
+
+        super().__init__(target, **kwargs)
+
+        self.scale = scale
+        self.current_point = self.initial_point
```

Comment:
For `pCN`, maybe it makes sense to draw a sample from prior as `initial_point`? It seems so, as the prior will always be Gaussian, and Gaussian comes with a `sample` method.

Reply:
#378